composer.json

```diff
@@ -15,7 +15,8 @@
         "php": "^5.5.9|^7.0",
         "symfony/framework-bundle": "^2.8|^3.0",
         "sonata-project/seo-bundle": "^1.1.3|^2.0",
-        "doctrine/collections": "^1.0"
+        "doctrine/collections": "^1.0",
+        "phpcr/phpcr-utils": "^1.2.9"
```

Comment:
maybe we should create a 2.0 for phpcr-utils by dumping php < 5.6? We can instantiate every array with [] them.